### ISSUE TITLE
Is there any way to set a custom location such as the location of Obsidianvault

### ISSUE THREAD

**Issue by skillhacker-code** [deathau/markdownload#114]
I would like to save the page downloaded using markdownload to go to a custom location in my pc Like Location of my obsidian vault , is there any way to do that .

I am currently using FreeFileSync to sync the file from Download/webclipping folder to obsidian webclipping folder

**Comment by JacSavage** [comment]
If you*re on Linux, you can add a symlink to your download folder that points to your vault

**Comment by 00dani** [comment]
Chiming in to confirm that @JacSavage's suggestion works perfectly on MacOS as well. I can't confirm whether it works on Windows, though.

**Comment by not-AMM** [comment]
As the [owner mentioned](https://github.com/deathau/markdownload/issues/21#issuecomment-701703402) in #21:

> The browser only allows me to specify whether I should open the Save As modal, or stick with the browser default. And I don't have any access to the file system to list folders, let alone write to one. It's a potential security issue for the browser.  

I think that extensions are limited to the `Downloads` folder (and subfolders) for security reasons. So there are no extensions that have access to any other folder. 

P.S.: This is an assumption since I couldn't find any information that could confirm this. In case I find information, I'll attach it here.

**Comment by 00dani** [comment]
I've finally booted my Windows machine again and I can confirm that putting a symlink in Downloads does the trick here too. Since Windows is weird, the commands I needed to do this looked like:
```cmd
cd ~\Downloads
mklink /D Obsidian ..\Nextcloud\Notes\!Inbox
```
The `/D` flag is required because Windows differentiates from symlinks to files and symlinks to directories, for… some reason.

Once done, MarkDownload works perfectly on my Windows box too, just like on MacOS or Linux. :cat:
